cmd/documentation/exchange_templates/exchange_websocket_readme.tmpl

```diff
@@ -26,27 +26,27 @@ Example setup for the `websocket` package connection:
 package main
 
 import (
-	"github.com/thrasher-corp/gocryptotrader/internal/exchange/websocket"
-    exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
-    "github.com/thrasher-corp/gocryptotrader/exchanges/request"
+	"github.com/thrasher-corp/gocryptotrader/exchange/websocket"
+	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/request"
 )
 
 type Exchange struct {
-    exchange.Base
+	exchange.Base
 }
 
 // In the exchange wrapper this will set up the initial pointer field provided by exchange.Base
 func (e *Exchange) SetDefault() {
-    e.Websocket = websocket.NewManager()
+	e.Websocket = websocket.NewManager()
```

Comment:
Thank you for appeasing my OCD, can sleep well tonight 😆

Reply:
🚀